nservicebus/architecture/publishing-from-web-applications.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Publishing Events from Web Applications
+summary: "Guidance on why you should or should not publish events from a web application"
+tags: []
+redirects:
+---
+
+You may have heard that [it's bad practice to publish events from a web application](http://www.make-awesome.com/2010/10/why-not-publish-nservicebus-messages-from-a-web-application/). This guidance is a bit outdated, as improvements Microsoft has made to IIS, more varied methods of hosting .NET web applications, and the addition of additional NServiceBus transports have made things much more nuanced.

```

Comment:
Do we need to link to your blog here? It feels weird to me that official docs use phrasing like "you may have heard". Should we just write what the guidance is as of now? (of course taking different transports into account as needed)